.github/workflows/daily-workflow-ci.yml

```diff
@@ -1,4 +1,4 @@
-name: Daily Workflow CI
+name: Report Generation CI
```

Comment:
Which report? Now that we are changing this it would be great to make sure the name reflects directly what the workflow is doing

Reply:
It would be: `Buildfarmer Log Daily Report Generation CI`

Reply:
Go for it and LGTM